### PR TITLE
Helpers

### LINKED ISSUE #1117: Integrate Existing Helpers View into Beta
- [ ] Create React Implementation of [helpers](https://www.mycrypto.com/helpers.html); integrate into beta

https://beta.mycrypto.com/helpers.html leads to a blank page.

In the footer there is a link to the helpers page, but it leads to the MyCrypto production version's helpers page and when you navigate off that, you end up on the prod. version of MyCrypto.

### PR DESCRIPTION
**Pull request by domalak**
Closes #1117 

### Description

This is the new Helpers page for MyCrypto.

### Changes

Created all sections that were in legacy Helpers page and adapted the look to MyCrypto design.

### Steps to Test

Use "Helpers & ENS Degugging" link in the footer to open the page.

### CODE REVIEW

**Comment by domalak** [comment]
hey @skubakdj thanks for review.
I'll start working on that.

**Comment by dternyak** [comment]
Hey @domalak 

Looks like we're getting really close to merging this. Do you mind fixing the conflicts in preparation for the merge?

**Comment by domalak** [comment]
hey @dternyak 
no problem. I've resolved conflicts. Turned out you reorganized actions, sagas and reducers into folder called features.
Please check now.

**Comment by dternyak** [comment]
Thanks @domalak !

File: common/containers/Tabs/Helpers/components/RecoverPK/index.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+                onChange={this.onChange}
+                isValid={isValidETHAddress(address)}
+                name="address"
+              />
+            </label>
+
+            <button
+              className="btn btn-primary"
+              onClick={this.searchPK}
+              disabled={!isValidPrivKey(pk) || !isValidETHAddress(address)}
```
Comment (skubakdj) [inline common/containers/Tabs/Helpers/components/RecoverPK/index.tsx]: Could change `address` to `address.toLowerCase()` so the button won't  be disabled if the user enters a checksummed address incorrectly.

File: common/features/ens/shaBid.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// import { ShaBidRequested, ShaBidAction, ShaBidSucceeded, ShaBidFailed } from 'actions/ens';
```
Comment (skubakdj) [inline common/features/ens/shaBid.ts]: unused import

**Comment by domalak** [comment]
Hey @skubakdj 
Sorry, forgot about this for some time.
So I fixed the most issues you mentioned in last comment.
Two exceptions:
1. in Convert Units I couldn't notice that numbers are changed to `e` notation if they're getting large.
2. in ENS Debug about `Start Auction And Bid Data` field. I think the legacy app calculation is incorrect. If you go to https://etherscan.io/ens and open any transaction with `startAuctionsAndBid` function, the input data is more close to our data than data in legacy field. So hopefully we're ok here.

**Comment by CryptoQueen628** [comment]
---

**Comment by tayvano** [comment]
Hey @domalak lmk if you want to re-sync ever on stuff. We appreciate all the work you've done since the early days on helpers, and i still use them to this day to troubleshoot things. Hope all is well with you.

**Comment by domalak** [comment]
hey @tayvano . Sorry for the late answer. I've been reflecting lately if I'm really into crypto or was just there for the hype. Turned out I'm still in cryptoworld but decided to start some own local bitcoin exchange here in my country.
As for the Helpers I'll gladly help with updating my changes. Not sure about new features, can't spare much time unfortunately.

**Pull request closed without merging**